ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -113,7 +113,7 @@ export default function PermissionsConnectPermissionList({ permissions }) {
           typeof listValue === 'function' ? listValue(permission) : listValue;
 
         return (
-          <div className="permission" key={label}>
+          <div className="permission" key={idx}>
```

Comment:
This seems strictly worse. Why use the index as the key? The key should uniquely identify the entry, so that React can tell the difference between permissions getting re-arranged v.s. their properties being changed. The label was already a unique representation of the permission (or so I thought). The permission name would work as well. But the index does not serve as an adequate key, it is not unique per permission.

Reply:
The reason I changed this to idx was because you can have multiple permissions returning as `UNKOWN_PERMISSION`, using that label as a key doesn't work. I ran into that problem when the key was as label before.

Reply:
Ahhh I see. Well, using an index silences the warning, but it doesn't fix the problem. The index is what React already falls back to internally.

Could we use the permission name instead maybe? That shouldn't be duplicated.

Also, which permissions were coming up as unknown? :thinking: They are supposed to all be listed here.

Reply:
I think it may have been part of the test-data, hmmm maybe we don't run into that problem in production since we're cognizant of all the permissions we allow...I'll change it to just `permission` then. Thanks!